src/models/user.ts

```diff
@@ -127,7 +127,7 @@ export class User extends Model {
   }
 
   eagerPrimaryEmailAddress () {
-    return new PrimaryEmailAddress(this._userData.primaryEmailAddress, this.sessionHandler)
+    return typeof this._userData.primaryEmailAddress !== 'undefined' ? new PrimaryEmailAddress(this._userData.primaryEmailAddress, this.sessionHandler) : undefined
```

Comment:
Don't use the `undefined` global. Make this a one line ternary like the other `eager` methods.

Reply:
Did you you mean to put this issue?  This is using the one line ternary.